### PR TITLE
Fix not a function errors

### PR DESCRIPTION
**Pull request by svechinsky**
Solves https://github.com/truffle-box/drizzle-box/issues/24
Due to webpack not resolving an import without a `export default`  drizzle was throwing:
TypeError: (0 , _drizzle.generateContractsInitialState) is not a function
When using the example from the drizzle box

### CODE REVIEW

**Comment by mnaamani** [comment]
@svechinsky  nice, I was planning to share the same fix. There is just one small issue here in that It breaks `generateStore` export, so in `src/generateStore.js`, change 

`import { generateContractsInitialState } from './generateContractsInitialState'`
to 
`import generateContractsInitialState from './generateContractsInitialState'`

**Comment by svechinsky** [comment]
I did, thank you!!!

File: src/generateContractsInitialState.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { generateContractInitialState } from './generateContractInitialState'
+import generateContractInitialState from './generateContractInitialState'
```
Comment (mnaamani) [inline src/generateContractsInitialState.js]: I think you meant to make this change in `generateStore.js`

File: src/generateStore.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import reducer from './reducer'
 import { generateContractsInitialState } from './generateContractsInitialState'
```
Comment (mnaamani) [inline src/generateStore.js]: this should be changed to
 `import generateContractsInitialState from './generateContractsInitialState'`

**Comment by svechinsky** [comment]
Thanks!!!

**Comment by mnaamani** [comment]
@svechinsky great, but you need to revert (undo) https://github.com/trufflesuite/drizzle/pull/95/commits/6c9dd025b50619e530b451a66791bb599784cbac

**Comment by mnaamani** [comment]
LGTM 👍

**Comment by hairychris** [comment]
This seems to have been fixed in the latest version, should this be closed now?

**Comment by DiscRisknBisque** [comment]
Thanks for tackling this @svechinsky ! Closing only because I went through the PRs in chronological order and an earlier one implemented this fix as well. The fix is present in release v1.2.2

**Pull request closed without merging**